src/styles/index.scss

```diff
@@ -0,0 +1,3 @@
+@import "./calendar.scss";
+@import "./reset.scss";
```

Comment:
import reset and variables should be at the top of this file bcs rest can use them so it is important to keep them in 1st place